src/redis_stack/working_with_graphs.md

```diff
@@ -93,4 +114,21 @@ GRAPH.QUERY bikes_graph 'MATCH p=(u1:User {Name: "Andrea"})-[f:FOLLOWS]->(u2:Use
 
 ```redis All users who I follow who reviewed this bike with more than 3 stars
 GRAPH.QUERY bikes_graph 'MATCH p=(u1:User {Name: "Andrea"})-[f:FOLLOWS]->(u2:User)-[r:REVIEWED]->(b:Bike {Model: "Hillcraft"}) WHERE r.Stars>3  return p'
-```
\ No newline at end of file
+```
+
+## Use Bloom to check if username is free 
+
+Wonder how a Bloom filter can be used for your bike shop? For starters, you could keep a Bloom filter that stores all usernames of people who've already registered with your service. That way, when someone creates a new account, you can very quickly check if that username is free. If the answer is yes, you still have to go and check the main database for the precise result. But, if the answer is no, you can skip that call and continue with the registration. 
```

Comment:
We shouldn't recommend using BF to check if something exist in Redis (in this case - if there is some node in RedisGraph). RedisGraph has the full information, so why use probabilistic data. Also, RedisGraph is stored in RAM so it should be fast enough for such qeries.